README.md

```diff
@@ -133,14 +133,142 @@ pipe = LTXPipeline.from_pretrained(
     "Lightricks/LTX-Video", torch_dtype=torch.bfloat16
 ).to("cuda")
 + pipe.load_lora_weights("my-awesome-name/my-awesome-lora", adapter_name="ltxv-lora")
-+ pipe.set_adapters(["ltxv-lora"], [1.0])
++ pipe.set_adapters(["ltxv-lora"], [0.75])
 
 video = pipe("<my-awesome-prompt>").frames[0]
 export_to_video(video, "output.mp4", fps=8)
 ```
 
 </details>
 
+<details>
+<summary> Hunyuan Video </summary>
+
+### Training:
+
+```bash
+#!/bin/bash
+
+# export TORCH_LOGS="+dynamo,recompiles,graph_breaks"
+# export TORCHDYNAMO_VERBOSE=1
+export WANDB_MODE="offline"
+export NCCL_P2P_DISABLE=1
+export TORCH_NCCL_ENABLE_MONITORING=0
+export FINETRAINERS_LOG_LEVEL=DEBUG
+
+GPU_IDS="0,1"
+
+DATA_ROOT="/fsx/aryan/datasets/dataset-cakify-yt"
+CAPTION_COLUMN="prompts.txt"
+VIDEO_COLUMN="videos.txt"
+OUTPUT_DIR="/fsx/aryan/models/training/ltxv-loras/ltxv_cakify_2000_1e-5_constant"
+
+# Model arguments
+model_cmd="--model_name ltx_video \
+  --pretrained_model_name_or_path Lightricks/LTX-Video"
+
+# Dataset arguments
+dataset_cmd="--data_root $DATA_ROOT \
+  --video_column $VIDEO_COLUMN \
+  --caption_column $CAPTION_COLUMN \
+  --id_token afkx \
+  --video_resolution_buckets 17x512x768 49x512x768 61x512x768 129x512x768 \
+  --caption_dropout_p 0.05"
+
+# Dataloader arguments
+dataloader_cmd="--dataloader_num_workers 0"
+
+# Diffusion arguments
+diffusion_cmd="--flow_resolution_shifting"
```

Comment:
```suggestion
diffusion_cmd=""
```

Reply:
Removing because I'm yet to test which is better since we don't know how exactly Hunyuan was trained